src/core/qgshelp.h

```diff
@@ -0,0 +1,70 @@
+/***************************************************************************
+  qgshelp.h
+  --------------------------------------
+  Date                 : December 2016
+  Copyright            : (C) 2016 by Alexander Bruy
+  Email                : alexander dot bruy at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSHELP_H
+#define QGSHELP_H
+
+#include <QtCore>
+
+/** \ingroup core
```

Comment:
I think this should rather go into gui than core (there is no QDesktopServices on server).